client/src/components/Articles.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import axios from 'axios';
+import '../App.css';
+
+export class Articles extends React.Component {
+
+    state = {
```

Comment:
you can leave comments like this if you have some suggestions

Reply:
Looks good, merge away!